app/models/payment_state.rb

```diff
@@ -40,25 +40,32 @@ def ttl_paid
   end
```

Comment:
status1_current_camp_payments scope might be used in line 39

Reply:
additional DB calls that are not necessary